lib/core/versions/0.7.0/OperationProcessor.ts

```diff
@@ -62,6 +64,27 @@ export default class OperationProcessor implements IOperationProcessor {
     return appliedDidState;
   }
 
+  public async getRevealValue (anchoredOperationModel: AnchoredOperationModel): Promise<Buffer> {
+    if (anchoredOperationModel.type === OperationType.Create) {
+      throw new SidetreeError(ErrorCode.OperationProcessorCreateOperationDoesNotHaveRevealValue);
+    }
+
+    const operation = await Operation.parse(anchoredOperationModel.operationBuffer);
+
+    let encodedRevealValue;
+    if (operation.type === OperationType.Recover) {
```

Comment:
nit: prefer to see switch statements.

Reply:
Personally I don't refer it, but since you asked, I will do it just for you @OR13!